### ISSUE TITLE
Blazor Server app CSS isolation issue

### ISSUE THREAD

**Issue by majorimi** [dotnet/aspnetcore#26144]
I'm testing the new CSS isolation feature announced [here](https://devblogs.microsoft.com/aspnet/asp-net-core-updates-in-net-5-preview-8/). It works great for Blazor WebAssembly applications. A bit annoying to always add this line but not a huge issue:
`<link href="_framework/scoped.styles.css" rel="stylesheet" />`

However when I'm trying to use the same component on a **Blazor Server App** CSS is not working and in the browser Console there is this error: 
`Failed to load resource: the server responded with a status of 404 () https://localhost:44307/_framework/scoped.styles.css`

Basically Stylesheet is not generated on Server Apps. Should it work or am I using it wrong?

**Comment by captainsafia** [comment]
@majorimi Are you able to reproduce this issue in a newly created project?

The location of the generated stylesheet was changed in RC1, so that might be causing your 404 errors here.

**Comment by BruceHunter** [comment]
I think you get that error if you haven't actually added at least 1 scoped css file for a specific component.

**Comment by majorimi** [comment]
I have a Razor Class Library project with a component in it. Which using <component>.razor.css isolated CSS file. I build that component and reference it into 2 Blazor Apps.

- Blazor WebAssembly App: added stylesheet into wwwroot/index.html and **component work perfectly CSS applied**.
- Blazor Server App: added stylesheet into Pages/_Hosts.cshtml and **component looks weird CSS not loaded got 404 in Browser console**.

So the weird thing is the same component with isolated CSS works with WebAssebly App but not working with Server side (ASP hosted + SignalR) apps.

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
This has been addressed in RC2 release already. Please update when it releases and your issue will go away.

**Comment by majorimi** [comment]
RC1 has a lot of issues... Hope you fix all of them and ship RC2 soon..
Thanks.

**Comment by majorimi** [comment]
@mkArtakMSFT I'm facing one more issue related to CSS. The above mentioned isolated CSS was working with WebAssembly projects when Razor Class Library project was referenced directly. But when it is packaged to Nuget and used via Nuget package got this error:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error		The scoped css file 'C:\Users\Major\.nuget\packages\majorsoft.blazor.components.loading\0.9.1-rc.1.20452.10\contentFiles\any\net5.0\LoadingPage.razor.css' was defined but no associated razor component was found for it.	Blazor.Components.TestApp (Samples\Blazor.Components.TestApp)	C:\Program Files\dotnet\sdk\5.0.100-rc.1.20452.10\Sdks\Microsoft.NET.Sdk.Razor\build\netstandard2.0\Microsoft.NET.Sdk.Razor.ScopedCss.targets	97	
```

**Comment by bmoteria** [comment]
For reference, you are required to have at least one scoped css file (MainLayout.razor.css) like @BruceHunter mentioned in order to avoid `404 - Failed to load resource` console error.
